foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/platform/database/PostgreSQLPlatform.java

```diff
@@ -252,7 +254,7 @@ public static ExpressionOperator regexpOperator() {
     /**
      * INTERNAL: Postgres to_number has two arguments, as fix format argument.
      */
-    protected ExpressionOperator toNumberOperator() {
+    private static ExpressionOperator toNumberOperator() {
```

Comment:
why not leaving this protected?

Reply:
There is no reason to have more than private on platform specific ExpressionOperator instances creators. But I'll leave this untouched.